GOVERNANCE.rst

```diff
@@ -384,27 +414,41 @@ If the proposed change cannot gather a 2/3 majority of the votes cast,
 the CC tries to resolve the deadlock.
 
 -  The CC will use consensus seeking.
--  If no consensus can be found within a month, the decision is made
-   through a simple majority vote (with tie breaking) among the CC
+-  If no consensus can be found within twenty working days excluding weekends
+   since the beginning of the stage-1 "reasonable time for consideration" period,
+   the decision is made through a simple majority vote (with tie breaking) among the CC
    members.
--  Any CC decision must be supported by an :ref:`steps`, which has been made public and discussed before the vote.
+-  If this vote is needed, the vote should take place within twenty-five working days after the beginning
+   of the stage-1 "reasonable time for consideration" period, during a regular, minuted CC meeting.
+-  Any proposal reaching stage 3 must be supported by an :ref:`steps`,
+   which has been made public at least 5 days, excluding week-ends, before the vote.
 
 .. _steps:
 
 sktime enhancement proposal
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For all decision of the CC, an sktime enhancement proposal (STEP) must have been made
-public and discussed before the vote.
+sktime enhancmenet proposals (STEPs) are required for:
 
-An sktime enhancement proposal is a consolidated document, with a concise
+* certain types of proposed changes, by default, see `decision making process <#Decision-making>`__
+* for all stage 3 decisions
+
+If a STEP is required by a vote, it must have been made public at least 5 working days (excluding week-ends) before that vote.
+
+A STEP is a consolidated document, with a concise
 problem statement, a clear description of the proposed solution and a
 comparison with alternative solutions, as outlined in our
 `template <https://github.com/sktime/enhancement-proposals/blob/master/TEMPLATE.md>`__.
 
-We collect and discuss proposals in sktime’s `repository for
+A complete STEP must always include at least a high-level design for the proposed change,
```

Comment:
As before, let's define STEPs in a single place in the README in the STEPS repo, so a link should do here.

Reply:
that would make that readme a formal document and an appendix to this one, are you sure?

Reply:
As said before, I don't see this as a legal document and I don't think it should be, so not really sure what "formal document" or "appendix" means, having a link should do the job

Reply:
I suggest we have the STEP guidelines on the website at: https://www.sktime.org/en/latest/enhancement_proposals.html

Reply:
So we can easily link it there

Reply:
yes, this is already linked, so I assume this is resolved.